install/configuring_inventory_file.adoc

```diff
@@ -643,6 +643,11 @@ in the `edits` field will update the related ConfigMap (`node-config-compute` in
 this example), which will ultimately affect the node's configuration file on the
 host.
 
+[NOTE]
+====
+Running the *_openshift_node_group.yaml_* playbook only updates new nodes. It cannot be run to update existing nodes in a cluster.
```

Comment:
In the future, a reference to a file name should just be in backticks, not this combo. But I see that others in this file use this same format, so it's fine for now.

Reply:
Good to know. Thank you!